### PR TITLE
3. Retrieving data as lists

### PR DESCRIPTION
**Pull request by mukesh51**
Updated Tutorial to go in flow with the previous sections. export was missing, selector was not correct. removed moduleId, which could have thrown errors for new users. Component name has been corrected.

### CODE REVIEW

File: docs/3-retrieving-data-as-lists.md [hunk 1]
```diff
@@ -54,21 +60,23 @@ const queryList = af.database.list('/items', {
...
 
 @Component({
-  selector: 'app',
-  templateUrl: `
+  selector: 'app-root',
+  template: `
   <ul>
     <li *ngFor="let item of items | async">
-      {{ item.name }}
+       {{ item.$value }}
```
Comment (katowulf) [inline docs/3-retrieving-data-as-lists.md]: I don't like `item.$value` here. This field is only useful for primitives and that's not the common use case. `item.name` represents an arbitrary value in the field. Let's put in something more useful for debugging and initial output? How about:  `{{ item | json }}`

File: docs/3-retrieving-data-as-lists.md [hunk 2]
```diff
@@ -54,21 +60,23 @@ const queryList = af.database.list('/items', {
 To get the list in realtime, create a list binding as a property of your component or service.
 Then in your template, you can use the `async` pipe to unwrap the binding.
 
+Update `/src/app/app.component.ts` to import `FirebaseListObservable` from angularfire2 and iterate thru the list once data is retrieved. Also note the change in attribute `templateUrl` to inline `template` below.
```
Comment (katowulf) [inline docs/3-retrieving-data-as-lists.md]: Why are we changing this to an inline template? If the angular-cli installation creates it as an external URL, I'd prefer to use that convention as it's clearly our recommended best practice. @deast opinions?

**Comment by katowulf** [comment]
A couple comments here to address before I merge. Great work again!

File: docs/3-retrieving-data-as-lists.md (hunk 1 above)
Comment (mukesh51) [inline docs/3-retrieving-data-as-lists.md]: updated to add {{ item | json }}

Yep this makes more sense.

File: docs/3-retrieving-data-as-lists.md (hunk 2 above)
Comment (mukesh51) [inline docs/3-retrieving-data-as-lists.md]: @katowulf Changing this from external Url to inline template is just to keep the flow of the tutorial consistent across all the sections. Since in the initial section, the changes were done to an inline template. Happy to revert the changes as per your steer. 

Note:- I am also planning to make changes to section 4 and 5 of the tutorial to keep it consistent, while making other changes.

**Comment by mukesh51** [comment]
@katowulf @davideast This is still open. Do i need to add anything more, before this can be approved.
